common/dynasties/24000_gilnean.txt

```diff
@@ -51,24 +51,24 @@
 24035 = {name = "Camus" culture = gilnean}
 24036 = {name = "Holmes" culture = gilnean}
 24037 = {name = "Prince" culture = gilnean}
-24038 = {name = "Thor" culture = gilnean}
-24039 = {name = "Storm" culture = gilnean}
+24038 = {name = "Norman" culture = gilnean}
+24039 = {name = "Williams" culture = gilnean}
 24040 = {name = "Harrison" culture = gilnean}
 24041 = {name = "Royston" culture = gilnean}
 24042 = {name = "Trollope" culture = gilnean}
 24043 = {name = "Vexx" culture = gilnean}
 24044 = {name = "Padley" culture = gilnean}
 24045 = {name = "Keeny" culture = gilnean}
 24046 = {name = "Snowdon" culture = gilnean}
-24047 = {name = "Satan" culture = gilnean}
+24047 = {name = "George" culture = gilnean}
```

Comment:
Since Lucifer is an actual name. As an American, I believe no one should be named George.

```suggestion
24047 = {name = "Lucifer" culture = gilnean}
```

Reply:
Asmodeus might even be more "gilnean-y"

Reply:
Mmmkay, used **_Asmodeus_**.